lldb_formatters/crystal_formatters.py

```diff
@@ -0,0 +1,63 @@
+import lldb
+
+class CrystalArraySyntheticProvider:
```

Comment:
Should this be in this repository?

Reply:
This is a helper for LLDB debugger for those who wants to use them. They are Crystal specific so this is why I put them here.

Reply:
I think i'd rather have it in `etc/lldb_crystal_formatters.py`.

Reply:
It would be better in `bin/lldb_crystal_formatters.py` - it's a program.
`etc` is more for configuration-related files, or even sometimes some like completion scripts.